Lib9c/Action/NotEnoughFungibleAssetValueException.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Globalization;
+using System.Numerics;
+using System.Runtime.Serialization;
+using Libplanet.Assets;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    public class NotEnoughFungibleAssetValueException : Exception
+    {
+        public NotEnoughFungibleAssetValueException(string message) : base(message)
+        {
+        }
+
+        public NotEnoughFungibleAssetValueException(string require, string current)
+            : this(
+                $"Aborted as the signer's balance ({current}) is insufficient to pay entrance fee/stake ({require}).")
```

Comment:
How about this for consistency?
```suggestion
                $"Aborted as the signer's balance is insufficient to pay entrance fee/stake: {current} < {require}.")
```